<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Aktualne Pomiary</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditTemp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAqi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTvoc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEco2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wilgotność</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ciśnienie atmosferyczne</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Natężenie dźwięku</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wskaźnik jakości powietrza</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Całkowite lotne związki organiczne</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ekwiwalent CO2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Temperatura</string>
     </attribute>
     <widget class="QWidget" name="widget_100" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_101" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Wilgotność</string>
     </attribute>
     <widget class="QWidget" name="widget_102" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_103" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ciśnienie</string>
     </attribute>
     <widget class="QWidget" name="widget_104" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_105" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Natężenie Dźwięku</string>
     </attribute>
     <widget class="QWidget" name="widget_106" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_107" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>AQI</string>
     </attribute>
     <widget class="QWidget" name="widget_108" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_109" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>TVOC</string>
     </attribute>
     <widget class="QWidget" name="widget_110" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_111" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>eCO2</string>
     </attribute>
     <widget class="QWidget" name="widget_112" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_113" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>731</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
